app/views/fragments/metricsHeader.scala.html

```diff
@@ -0,0 +1,23 @@
+@(oldInstancesCount: Int, totalInstancesCount: Int, agePercentiles: utils.Percentiles)
+
+@percentileRow(percentileKey: Int, percentileValue: Int) = {
```

Comment:
Should this go in the [fragments](https://github.com/guardian/amiable/tree/master/app/views/fragments) package?